assets/build/js/onetap.js

```diff
@@ -0,0 +1,22 @@
+window.LoginWithGoogleDataCallBack = function( response ) {
+    var xhr = new XMLHttpRequest();
+    xhr.open('POST', TempAccessOneTap.ajaxurl, true);
+    xhr.setRequestHeader('Content-Type', 'application/x-www-form-urlencoded');
+    xhr.onreadystatechange = function() {
+        if(xhr.readyState === XMLHttpRequest.DONE) {
+            var status = xhr.status;
+            if ( status === 200 ) {
+                var response = JSON.parse( xhr.responseText );
+
+                if ( ! response.success ) {
+                    alert( response.data );
+                    return;
+                }
+
+                window.location = response.data.redirect;
```

Comment:
Is it possible to validate redirect url to make sure it's not redirecting to any other site url?

Reply:
Why is this build file not minified? Please address the feedback in src file.